packages/orchestration/test/examples/swapExample.test.ts

```diff
@@ -27,13 +28,7 @@ test('start', async t => {
     installation,
     { Stable: ist.issuer },
     {},
-    {
-      agoricNames: bootstrap.agoricNames,
-      localchain: bootstrap.localchain,
-      orchestrationService: bootstrap.orchestration,
-      storageNode: bootstrap.storage.rootNode,
```

Comment:
have we been passing the whole storage root to the contract? that seems weird.

(not a request for change)